.travis.yml

```diff
@@ -16,9 +15,15 @@ env:
 
 matrix:
     include:
-        - php: 5.5
```

Comment:
Same here, we do want to test on both the lowest supported Symfony and PHP versions.